service/synapse-service-rest/src/main/java/io/americanexpress/synapse/service/rest/controller/reactive/BaseDeleteReactiveController.java

```diff
@@ -39,10 +41,10 @@
     @DeleteMapping("/{identifier}")
     @ResponseStatus(HttpStatus.NO_CONTENT)
     @Operation(tags = "Delete Operation", summary = "Deletes a resource reactively")
-    public Mono<ResponseEntity<Void>> delete(@PathVariable String identifier) {
+    public Mono<ResponseEntity<Void>> delete(@RequestHeader HttpHeaders headers, @PathVariable String identifier) {
         logger.entry(identifier);
 
-        var serviceResults = service.delete(identifier);
+        var serviceResults = service.delete(headers, identifier);
         var responseEntity = serviceResults
                 .map(res -> new ResponseEntity<Void>(HttpStatus.NO_CONTENT))
                 .defaultIfEmpty(ResponseEntity.notFound().build());
```

Comment:
I fixed this in my PR but delete line 50 if possible

Reply:
You should be getting 204 now after deleting line 50

Reply:
done